test/ffmpeg-qsv/transcode/transcoder.py

```diff
@@ -76,6 +79,13 @@ def validate_spec(self):
       codec = output["codec"]
       mode  = output["mode"]
       assert mode in ["sw", "hw"], "Invalid test case specification as output mode type not valid"
+      width = output.get("width", 0)
+      height = output.get("height", 0)
+
+      if width > 0 and height > 0:
+        if not (self.mode == "hw" and mode == "hw"):
+          slash.skip_test("incorrect mode, scaling not enabled")
```

Comment:
This should be a test failure (i.e. `assert`) since it is a user spec violation.

But this also begs the question... why can't we support sw scaling?

Reply:
removed this check and enabled SWtoHW scaling.